controller/src/main/java/io/pravega/controller/metrics/StreamMetrics.java

```diff
@@ -243,6 +204,42 @@ public void deleteStream(String scope, String streamName, Duration latency) {
         deleteStreamLatency.reportSuccessValue(latency.toMillis());
     }
 
+    /**
+     * This method reports the latency of delete Stream event processing.
+     *
+     * @param latency       Latency of the deleteStreamEvent operation.
+     */
+    public void deleteStreamEvent(Duration latency) {
+        deleteStreamEventLatency.reportSuccessValue(latency.toMillis());
+    }
+
+    /**
+     * This method reports the latency of update Stream event processing.
+     *
+     * @param latency       Latency of the updateStreamEvent operation.
+     */
+    public void updateStreamEvent(Duration latency) {
+        updateStreamEventLatency.reportSuccessValue(latency.toMillis());
```

Comment:
Should these resources be closed? I see a `reset()` method that closes them, but you missed to add these new metrics.